docs/site/pages/plugins/async-node.mdx

```diff
@@ -34,6 +34,11 @@ The `AsyncNodePlugin` exposes an `onAsyncNode` hook on all platforms. The `onAsy
 
 User should tap into the `onAsyncNode` hook to examine the node's metadata before making a decision on what to replace the async node with. The return could be a single asset node or an array of asset nodes.
 
+### Improvements
+
+The AsyncNodePluginPlugin class is now capable of handling the below :
+- The resolvedMapping map now sets the node.id as the key and either parsedNode or node as the value, depending on whether parsedNode is truthy.
+- This means that the plugin is now capable of resolving the async node even when the consumer returns a null/undefined value.
```

Comment:
Rather than document the implementation details, can we document what it enables? Maybe something like:

```suggestion
### Edge cases

If for some reason, the process for resolving some `AsyncNode` fails, or is no longer relevant, you can resolve the promise for that `AsyncNode` with `undefined` or `null` to replace that node with "nothing"...
```

Reply:
If you want to document the implementation changes, you should put those in the release notes for this PR

Reply:
but we are actually returning the existing node itself instead of nothing

Reply:
Example for adding release notes:
https://github.com/player-ui/player/pull/296#issue-2124105491

Reply:
should we be more intentional about the use case, rather than stating this to be a fallback option of sort

Reply:
is this a question to me or @sugarmanz ?

Reply:
both, right now it's framed as a fallback option, I'm thinking we make this more intentional because i'm thinking about the future node removal work

Reply:
I think that's covered with:
> or is no longer relevant

But we could have an explicit use case around using async node to power toasts, or something like that.